repos/fdbt-site/tests/components/ProductGroupCard.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+import ProductGroupCard from '../../src/components/ProductGroupCard';
+
+describe('ProductGroupCard', () => {
+    it('should render not checked', () => {
+        const wrapper = shallow(
+            <ProductGroupCard
+                groupDetails={{
+                    id: 1,
+                    productIds: ['1', '2', '3', '4'],
+                    name: 'The capped products',
```

Comment:
can we rename to Product Group card 1 or something like that

Reply:
Why?

Reply:
Generally, we give the name as the component related name